diff --git a/utils/create_env.py b/utils/create_env.py
index f263961..5aee7fa 100644
--- a/utils/create_env.py
+++ b/utils/create_env.py
@@ -43,7 +43,7 @@ def make_iglu(*args, **kwargs):
    # starting_grid, relief = make_relief_map()
    # starting_grid = list(zip(*starting_grid))
     #print(len(starting_grid))
-    env = GridWorld(custom_grid, render=False, select_and_place=True, max_steps=1000)
+    env = GridWorld(custom_grid, render=False, select_and_place=True, max_steps=2000)
    # env.initialize_world(starting_grid, initial_position)
     env = Multitask(env, make_holes=True)
     # env = SelectAndPlace(env)
